plugin/documents.py

```diff
@@ -483,8 +483,13 @@ def _on_code_actions(self, responses: CodeActionsByConfigName) -> None:
             scope = 'region.yellowish lightbulb.lsp'
             icon = 'Packages/LSP/icons/lightbulb.png'
         else:  # 'annotation'
-            suffix = 's' if action_count > 1 else ''
-            code_actions_link = make_command_link('lsp_code_actions', '{} code action{}'.format(action_count, suffix))
+            if action_count > 1:
+                title = '{} code actions'.format(action_count)
+            else:
+                title = next(iter(responses.values()))[0]['title']
```

Comment:
Noticed a potential issue. With two or more servers running on a view, the dict can look like this:

```
{
   'LSP-eslint': [],
   'LSP-typescript': [{...}]
}
```

In which case this code can crash since the first key has no items.

Reply:
> Noticed a potential issue. With two or more servers running on a view, the dict can look like this:
> 
> ```
> {
>    'LSP-eslint': [],
>    'LSP-typescript': [{...}]
> }
> ```
> 
> In which case this code can crash since the first key has no items.

It only works when there is one `code action`
https://github.com/sublimelsp/LSP/blob/0bb5932bb9de33a68188ae790a871cf592088464/plugin/documents.py#L472-L475

Reply:
There can still be just one code action in the case I've mentioned.

Reply:
How about `next(itertools.chain.from_iterable(response.values()))["title"]`

Reply:
> How about `next(itertools.chain.from_iterable(response.values()))["title"]`

![image](https://user-images.githubusercontent.com/49985266/119624245-85149a80-be3b-11eb-810e-06e1e2a8ceba.png)

Reply:
> How about `next(itertools.chain.from_iterable(response.values()))["title"]`

Nice, it is very good